utils/rpms/Makefile

```diff
@@ -27,13 +27,11 @@ include packaging/Makefile_packaging.mk
 
 PACKAGING_CHECK_DIR ?= ../../../rpm/packaging
 
-$(NAME)-$(VERSION).tar.gz: $(shell git ls-files :/:)
+$(NAME)-$(VERSION).tar.gz: $(shell git ls-files --recurse-submodules :/:)
```

Comment:
Perhaps whether we specify "recursiveness" here should match whether we do so in archive.sh? E.g., we should either add `--recursive` after the `foreach` in archive.sh or remove `--recurse-submodules` here.